dist/yoast-seo.js

```diff
@@ -2151,35 +2151,25 @@ YoastSEO.ScoreFormatter.prototype.outputOverallScore = function() {
 };
 
 /**
- * retuns a string that is used as a CSSclass, based on the numeric score or the NA string
+ * Retuns a string that is used as a CSSclass, based on the numeric score or the NA string.
  * @param score
```

Comment:
This needs `{number|string}` and an explanation. I don't like the fact it can both accept an number and a string, but we'll ignore that for now.